src/Uuid/Uuid.php

```diff
@@ -146,7 +146,7 @@ public static function uuid_is_valid($uuid)
             return null;
         }
 
-        return null !== self::uuid_parse_as_array($uuid);
+        return (bool) self::isValid($uuid);
```

Comment:
let's cast in bool in `isValid` instead. It feels more natural to me that `isValid` returns a bool

Reply:
updated